acceptance-tests/src/test/resources/features/site/datasets.feature

```diff
@@ -73,6 +73,10 @@ Feature: As a consumer I need to be able to navigate to publication data sets
         Given I navigate to the "upcoming publication dataset" dataset page
         Then I should see the page not found error page
 
+    Scenario: Hide dataset if it doesn't have a parent publication
+        Given I navigate to the "dataset without publication" dataset page
+        Then I should see the page not found error page
```

Comment:
Would read slightly better if 'page not found' were embraced in quotes. (also applies to the scenario above).

Reply:
Fixed 😄